demo/cache/cache.html

```diff
@@ -1,61 +1,93 @@
-<!-- Page Content -->
+<uib-accordion close-others="oneAtATime"> <!-- FDP caching config -->
+<div uib-accordion-group class="panel-info">
+	<uib-accordion-heading> <span
+		class="glyphicon glyphicon-cog"></span> FDP cache configuration <i
+		class="pull-right glyphicon"
+		ng-class="{'glyphicon-chevron-down': status.open, 'glyphicon-chevron-right': !status.open}"></i>
+	</uib-accordion-heading>
+	<form>
+		<div class="form-group row" ng-repeat="fdp in fairDataPoints">
+			<div class="col-lg-4">
+				<div class="input-group">
+					<span class="input-group-addon">String</span> <input type="text"
+						class="form-control" aria-label="..." placeholder="Enter FDP name"
```

Comment:
If you are planning to use the accessibility attributes, enter a sensible description. Otherwise you might as well leave them out.